### PR TITLE
Your Benefits page - updated styles

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
## What does this pull request do?

LGA-884
Updates the page to use the new GDS styles for tickboxes.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_public/templates/macros/form.html [hunk 1]
```diff
@@ -317,8 +317,11 @@
   {% if input_type == 'radio' %}
     <div class="govuk-radios">
     {% for option in field %}
+      {% if option.data in separated_options %}
+        <div class="govuk-radios__divider">or</div>
+      {% endif %}
       <div class="govuk-radios__item">
-        <input class="govuk-radios__input" id="{{option.id}}" type="radio" name="{{field.id}}" value="{{option.value}}" >
+        <input class="govuk-radios__input" id="{{option.id}}" type="radio" name="{{field.id}}" value="{{option.data}}" >
```
Comment (said-moj) [inline cla_public/templates/macros/form.html]: Why the change from option.value to option.data and what's the difference

Comment (MalcolmVonMoJ) [reply cla_public/templates/macros/form.html]: option.value was an error introduced during my last change - so I fixed it here.  Should always have been option.data
